locale/en/messages.json

```diff
@@ -1117,6 +1125,7 @@
   "You are about to {0} {taskCountContent}. Are you sure you want to continue?": "",
   "You can also join us on our <0>Slack channel</0> or send us an email at <1>{0}</1>.": "",
   "You can configure the placement of agent nodes in regions and zones for high availability or to expand capacity to new regions when necessary": "",
+  "You can configure the placement of agent nodes in regions and zones for high availability or to expand capacity to new regions when necessary.": "",
```

Comment:
This is the same message as the previous one, but with a dot.

Reply:
https://github.com/mesosphere/dcos-ui-plugins-private/pull/982/commits/fd0712895419dfd4abde81a188f35b37229b1429